cli/azd/internal/appdetect/spring_boot.go

```diff
@@ -379,6 +393,26 @@ func IsValidDatabaseName(name string) bool {
 	return re.MatchString(name)
 }
 
+func detectPropertySpringCloudStreamBindingDestination(azdProject *Project, springBootProject *SpringBootProject) {
+	result := getBindingDestinationMap(springBootProject.applicationProperties)
+	for key, value := range result {
+		newKey := fmt.Sprintf("spring.cloud.stream.bindings.%s.destination", key)
+		azdProject.Metadata.BindingDestinationInProperty[newKey] = value
+	}
+}
+
+func detectPropertyEventhubsCheckpointStoreContainer(azdProject *Project, springBootProject *SpringBootProject) {
+	result := make(map[string]string)
+	for key, value := range springBootProject.applicationProperties {
+		if strings.HasSuffix(key, "spring.cloud.azure.eventhubs.processor.checkpoint-store.container-name") {
```

Comment:
The property prefix need to be updated:

> ![image](https://github.com/user-attachments/assets/d9fe7876-7a94-40fc-9ed1-b6cdfe125c1a)

Reply:
This is for checking the suffix, so no need to do it.